test/analytics/tests/analytics_from_the_server.ts

```diff
@@ -5,11 +5,10 @@
  * in compliance with, at your election, the Elastic License 2.0 or the Server
  * Side Public License, v 1.
  */
-
 import expect from '@kbn/expect';
-import type { TelemetryCounter } from '@kbn/core/server';
-import { Action } from '@kbn/analytics-plugin-a-plugin/server/custom_shipper';
-import { FtrProviderContext } from '../services';
+import type { Event, TelemetryCounter } from '@kbn/core/server';
```

Comment:
`Event` is a dom type, and typescript defaults to using that. We need to import the EBT `Event` here.